controller/argument_value_resolver.rst

```diff
@@ -146,10 +150,10 @@ and adding a priority.
 
         # app/config/services.yml
         services:
-            app.value_resolver.user:
-                class: AppBundle\ArgumentResolver\UserValueResolver
-                arguments:
-                    - '@security.token_storage'
+            # ...
+
+            AppBundle\ArgumentResolver\UserValueResolver:
+                # arguments is not needed if you have autowire above under _defaults
```

Comment:
I personally prefer having it everywhere, this would limit situations hard to debug for new comers.